### PR TITLE
*: vendor tool migrate to glide

### PR DESCRIPTION
**Pull request by tiancaiamao**
See https://github.com/pingcap/tidb/issues/1645

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  - encoding/charmap
+  - encoding/internal
+  - encoding/internal/identifier
+  - encoding/japanese
+  - encoding/korean
+  - encoding/simplifiedchinese
+  - encoding/traditionalchinese
+  - encoding/unicode
+  - transform
+- package: google.golang.org/grpc
```
Comment (overvenus) [inline glide.yaml]: If `glide update` failed, please refer to https://github.com/pingcap/pd/blob/master/glide.yaml#L185

**Comment by shenli** [comment]
Please make sure that tidb-test could run and others could use TiDB as a library.

**Comment by tiancaiamao** [comment]
vendor directory of sub package is removed, and grpc is the now same version with pd
@overvenus 

compatible with previous _vendor/vendor, tidb-test is not affected.
@shenli

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@overvenus PTAL

**Comment by overvenus** [comment]
LGTM

**Pull request merged**